controller/monitor.go

```diff
@@ -251,7 +251,7 @@ func (m *MonitorMgr) checkCond(am *appMon) error {
 				if len(parts) != 2 {
```

Comment:
i believe this check here will bypass the code below since now there is one extra part so this needs to change to
if len(parts) != 3